.github/workflows/ci.yml

```diff
@@ -31,7 +31,9 @@ jobs:
     - name: Run Ruff linting check
       run: poetry run ruff check .
     - name: Run precommit checks
-      run: poetry run pre-commit run --all-files
+      run: |
+        export SKIP=no-commit-to-branch
```

Comment:
FWIW, it's more portable to specify this as
```
env:
  SKIP: no-commit-to-branch
```

Reply:
Thank you for the input - I've raised https://github.com/blockwork-eda/blockwork/pull/77 with the correction